### DISCUSSION TITLE
Change default message of merge commit to `PR tilte and description`

### DISCUSSION THREAD

**Discussion by eitsupi** [PRQL/prql#2361] [Ideas]
Currently, the merge commit message contains a list of messages from the squashed commit.
But given that the squashed commits may contain ultimately irrelevant material that was changed along the way, I think that PR's description would often be more intelligible to a later reader than commit message list.

So I am wondering if it is worth changing the commit message to `PR tilte and description`.
![image](https://user-images.githubusercontent.com/50911393/229280818-a5d20b42-ab35-4740-b01f-dad5cbdc2996.png)

**Comment by max-sixty** [comment]
+1 from me...

**Reply by max-sixty** [reply]
I switched it; we can reopen if anyone disagrees...
